docs/notebook_tutorial/notebook_classes.py

```diff
@@ -1,65 +1,2161 @@
-import os
```

Comment:
This file holds classes used by notebook. Could you check why this file has been edited? Also, please check all other edited files unrelated to the notebook you intended to edit.